src/ophys_etl/transforms/mesoscope_2p/tiff.py

```diff
@@ -262,11 +265,21 @@ def volume_scans(self):
 
     @property
     def plane_stride(self):
-        return len(self.plane_scans)
+        stride = 0
+        # only increment stride for unique zs, because same z end up in the
+        # the same page of the tiff - or at least did when I last tested
+        # that scenario in Scanimage
+        for z in np.unique(self.plane_scans):
+            if any([roi for roi in self.rois if roi.scanned_at_z(z)]):
+                stride += 1
+
+        return stride
 
     @property
     def volume_stride(self):
         stride = 0
+        # this logic is probably incorrect if there are 2+ volume scans
```

Comment:
is this a real situation?
if not, please remove this comment.
If so, either resolve with the science team, or add a warning.

Reply:
I emailed Natalia about this (the comment is from Jed's original PR).

Her response:
"This is a human error, operators are instructed to never have two planes at exactly the same depth."

which doesn't mean that it is impossible.

I asked her if we should throw an error or try to figure out what to do in this scenario (in which case, we would need some example data to examine). She said it would be helpful if the code could one day address this case, and she would be able to generate example data once things settle down on the data release.

None of which is a straightforward answer to your question. I just wanted to provide this for context.

Reply:
I propose raising an `Exception` then.

Reply:
I added this as an assertion in the `volume_scans` property method. It's a little gross, where I just double loop over the list and make comparisons. If there's a better or nicer way, please let me know.